src/main/resources/addons/patterns.json

```diff
@@ -0,0 +1,84 @@
+{
+  "patterns" : [
+    {
+      "patternId" : "threadsafety-threadsafety",
+      "level" : "Warning",
+      "category" : "ErrorProne"
+    },
```

Comment:
Why do you need this file?
It is not read by Codacy, and you have already the pattern inside the main `pattern.json` file.

Reply:
This is the manual documentation of addons. With addons, we can't generate documentation automatically, unfortunately. So this file will then be merged with the automatic generated files.

Reply:
Should these files be added to the tool image? I don't know if there is a better way to add them

Reply:
No the files are not added to the image, only the final files, inside `docs/`.

Reply:
I think it makes sense to have a base file for manual patterns, and then to join with patterns.json when generating the documentation. 👍